### ISSUE TITLE
connection stuck on an infinite loop

### ISSUE THREAD

**Issue by SebastianHa2** [jsforce/jsforce#1263]
jsforce seems to be creating a login loop for some reason for me

`    context.conn = new jsforce.Connection({loginUrl :"https://login.salesforce.com",
    instanceUrl: 'https://UM5.salesforce.com'
    });

    const username = "xxxxxxx";
    const securityToken = "xxxxxxxxxxx";
    const password = "xxxxxxxxx" + securityToken;

    context.conn.login(username, password, function(err, userInfo) {
      if (err) { return $log(err.message); }

      $log(context.conn.accessToken);
      $log(context.conn.instanceUrl);
      $log("User ID: ");
      $log(userInfo.id);
      $log("Org ID: ");
      $log(userInfo.organizationId);

    });`
    
    I've tried with and without the instanceUrl, there is no error, the info in the callback gets logged once, but in salesforce I can see hundreds of SUCCESSFUL log ins everytime
    
    
![image](https://user-images.githubusercontent.com/66742717/183112042-c08fda8e-47be-4d76-9e20-2d8898652238.png)

**Comment by BinyominCohen** [comment]
Same issue for me

**Comment by dale-vendia** [comment]
I've noticed this issue can happen when you need to reset your password or log in requires some manual action.

**Comment by SebastianHa2** [comment]
I've worked with the salesforce support and found out the issue was in the salesforce user setup for the API user. Below message how to fix it:

_"Thank you for giving time over call. As discussed, the "Session Security Level Required at Login" at the profile (System Administrator) level of the user - Tangle.io Development was set to "High Assurance" and as per recommendation, API users or Integration users are not supposed to be assigned to a profile which has session settings to - High Assurance. 

In our case as soon as we changed the Tangle.io Development user's profile to End User which has the "Session Security Level Required at Login" set to "none" and ran the process from Tangle and from Postman, we did not get any error and we started seeing responses back to us. 

Hence I would request you to create a new custom profile for your API user and tick the "API only user" permission on that profile and assign the integration/api user that you are going to use for this integration to that profile. This profile should also_ have "Session Security Level Required at Login" set to "none".

Refer document to understand more about this setting in terms of callouts and async processing - https://help.salesforce.com/s/articleView?id=000365609&type=1 "_
